components/MilestoneTimeline.tsx

```diff
@@ -88,8 +88,11 @@ const MilestoneTimeline = ({ workflow }: Props): React.ReactElement => {
             />
           </svg>
           <h3 className="lbh-body">
-            Authorised by{" "}
-            {workflow?.panelApprover?.name || workflow.panelApprovedBy}
+            Sent to{" "}
```

Comment:
this might be better off as `Authorised and sent to x by...`

Reply:
Coooool, I've updated!